content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -52,7 +52,7 @@ handler must be a valid [DNS label name](/docs/concepts/overview/working-with-ob
 
 ### 2. Create the corresponding RuntimeClass resources
 
-The configurations setup in step 1 should each have an associated `handler` name, which identifies
+The configurations set up in step 1 should each have an associated `handler` name, which identifies
```

Comment:
```suggestion
The configurations setup in step 1 should each have an associated `handler` name, which identifies
```

Reply:
This was one that I had thought about most extensively before fixing, and I recognize that just fixing the verb did not make it less awkward.  I think that this rephrasing makes it easier to read:

"The configurations that were set up in step 1 should each have an associated `handler` name, which identifies..."

Reply:
I added a commit with that rephrasing.